src/Ztm.Zcoin.NBitcoin.Tests/ZcoinTransactionData.cs

```diff
@@ -0,0 +1,35 @@
+        /// </summary>
+        public const string SigmaSpend =
+        "0100000001000000000000000000000000000000000000000000000000000000000000000001000000fd2805c49fd386cff7375c173de731d2311ada3339c6b328adc6c0100589dbadf16a7f140100142e8576afc3a601d219a07ffe0109320b1ddfdc742398f5521f3dd5ef65e1460000b4353941ca3d849c7a77c7a2cf7b2ee8c240c8a0f75c1fa5c83562f3094eae970000647199b6253ac2ebb447679ed5f5214ca9a4c5c8cb715bf15df102e333401455000015d7bba3a790b33003c83c00deb20ebb108fb5157923f4eb1dc74a3e0f635eec3897178ff0108ca7e3c4472496de5a35ef0304c438183a6b95adc7efc3ec635a47f92e66b522bea991bbd5211334ec9e8b76751dbb083af07af0ab6ba316a208d35d6fcf6ffe1801c0e24f0fb5c855d641d55b2c419100883b5f2361c93e589cb0ea4ba8bc27a5ffbb247b66fa5c492ecf2aaa2f10dc465878e6c36d2e64cc41e19cb8c66cd2c5aedfb56ca0dfd1bf0dd0fe108d397144cbef9404dd733276fc5a7d8d2822e4fb26712b5bfae47de8af29c250986b8fc12e4353f00b51d181ba2510277f5b7d11295d62434726f23d23138d10dc0896a139a9de57d343b510ba736c5be3baa21514f834b3843a70a88f1c13e96cfc65fb7ba247938de41485e1bdb01a8e10754129cbff966871018e4a0d5724fa8953bfcd0ca9f0b4f75e7c34bfbe60461b26760d0ba54d310ba74e4abcb97dd04f0d64fd32c379a5688d9f48d96eb8fd0c401b4524d7c66cb9d902680eb0f728f3fd5c82a615ac616c152797ecaa417dc40784f0eac76f71ca3a45e1939628bd9ac4357dcd54dd63310a4411e937ab67dc68d22ca3d9a8f3347d4d8ab46bcc34dd88d8f13c7a2507d898e8d84642b995476729efdbe355435eea9d1eea921a191eb2563f3d28cf3b7e48a3236de10479c50fe945bd084a7ee35f1746c0a038e9712a4e4ea7be126399d10141334ed890f5ab81aa5914773bd7c4b519dd31f3b5ad0555d10095305cf450767eb018f73052c195e5139a11def105bb996e060a8090f46e2bdb8fa0bd3e2bb368aaff4b95afcf1477a87d7a95b8fe95af3808b2001f9eb414e062a2b24aca15be3df8ae4bfb8a9b7ae8c77dbed9fc065f6442cc30711bec31c121f680f8c0c7d01e5ff7ed8e7ad54712f4ee6b3a2be8c5bf304b0b7b641d4c6f5aadd4134b642c4d1514087fa4c4c03beb3ccccf85700c95faaed29546c3904f7036e87c95fa233be210d3a3a4578dc9bb67d27478ea461b6277d1ddc5ecf916ed3606825ae117e607fe79b54e6bc0cb145388ff499423641cae14a566830d68fcee32121625448aeb0000a55feb8d84eaec780f6765eaa4c98f48596afab45af23a881b51624e3722afd800000e40316711dc08ac851e0a29be9d4e46925552398478649dfd618883deff676c0100debe4dec6a4b18dcdce7cf18bae189f7f8da78ad45da4b3793227abcfddea05e00001108494d428f5c87816d65abdbf49c737531d52724b3d021bb99471a0620fafa00007ce0de242f4f6ba5f51b59deecc58988ccd49d86eba250ee7d9f158e8064d3590000d236390dbe7b02ad2aec52920a7661bd0e92d1dbcea2e43735d348df42a91d4500009f25e3b6b1f63f5e70f096bb348a2affe838219a848a8e847f5d57dd49524044cf669d470ece838ce6aed424b6604d27b3dc64b9e3751657ccb3227327452d2a1e000000404b4c000000000053218d7c14b5b548618a3c1d6495f1308b8a53f1802510ae23a6ac28f573c5652102acaa897e87fc685a2fafd25ad9088d7d9ade785b4ecceaec8ff7760e7bde4e85404c9bf648ee8fe2d7ada7dd39583aac1ae7a352e6b2229a1a3da1c8d772c4e1024ff80f7addf5541abae2d642d432ebe4e362552a6d385e34a8aab833a75d9dd9ffffffff01c1454c00000000001976a914384bd481eb407570a867c106e2f5b7b39b51ee3488ac8c240300";
+
+        /// <summary>
+        /// Regtest Zerocoin remint transaction
```

Comment:
All others is mainnet data so this one should be mainnet too.

Reply:
I can't find a remint transaction on mainnet.